packages/common/src/components/ActionServiceDropdown/ActionServiceDropdown.tsx

```diff
@@ -67,12 +67,25 @@ const ActionsAsDropdown = ({
   const history = useHistory();
   const [isActionMenuOpen, setIsActionMenuOpen] = useState(false);
   const isPlain = variant === 'kebab';
+
+  // define once for all onToggle handlers
```

Comment:
```suggestion
  /**
 * @typedef {Object} ToggleEvent
 * @description Represents the possible event types that can be used for toggling actions.
 * 
 * @property {Event} Event - A standard DOM event.
 * @property {React.KeyboardEvent<Element>} React.KeyboardEvent - A React-specific keyboard event.
 * @property {React.MouseEvent<Element, MouseEvent>} React.MouseEvent - A React-specific mouse event.
 * @property {React.ChangeEvent<Element>} React.ChangeEvent - A React-specific change event.
 */
```

the `define once for all onToggle handlers` comment I suggested in my previous review was to help clarify why we need this type definition, when we actually write the code, we can do better. p.s. you did not use in in other onToggle handlers, please do use in in  all onToggle handlers ( see https://github.com/kubev2v/forklift-console-plugin/pull/1221/files#r1665219359 )

Reply:
I didn't like this fix since it manipulates the callbacks function signatures, but if you insist and think it is nicer then ok. I made the change in all relevant places now.